task/mgmt-gateway/src/main.rs

```diff
@@ -215,7 +228,6 @@ impl UsartHandler {
 
 struct NetHandler {
     net: Net,
-    rx_buf: [u8; Request::MAX_SIZE],
```

Comment:
Optional: if you liked having the member variable, you could do something like `rx_buf: &'static mut [u8]` (then pass the static buffer to the constructor as a slice)